fio.h

```diff
@@ -628,7 +628,7 @@ static inline bool multi_range_trim(struct thread_data *td, struct io_u *io_u)
 	return false;
 }
 
-static inline bool should_fsync(struct thread_data *td)
+static inline bool should_fsync(const struct thread_data *td)
```

Comment:
Seems like a random change that's not related to the fix.

Reply:
It's random in that it doesn't seem related to your changes, other than that you made get_sync_ddir() take a constant 'td' and then this one needs it too. But it's not related to the fix.

Making 'td' const in more spots is a worthwhile endeavor, but should be a separate thing from a fix.